### PR TITLE
feat: high-contrast theme

### PR DESCRIPTION
**Pull request by juliajforesti**

### CODE REVIEW

File: apps/meteor/client/sidebar/header/hooks/useAccountItems.tsx
Comment (dougfabris) [inline apps/meteor/client/sidebar/header/hooks/useAccountItems.tsx:1]: What about replacing the `useRoute` with `useRouter`?

File: apps/meteor/client/views/account/themes/ThemePage.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+import HighContrastUpsellModal from './HighContrastUpsellModal';
+import { themeItems as themes } from './themeItems';
+
+const ThemePage = () => {
+	const t = useTranslation();
+	const setModal = useSetModal();
+	const dispatchToastMessage = useToastMessageDispatch();
+	const { data: license } = useIsEnterprise();
+
+	const themePreference = useUserPreference<'light' | 'dark' | 'auto' | 'high-contrast'>('themeAppearence') || 'auto';
```
Comment (dougfabris) [inline apps/meteor/client/views/account/themes/ThemePage.tsx:18]: What about isolating the theme options in a type to use across the application? `type ThemeOptions = 'light' | 'dark'` etc

File: apps/meteor/public/images/high-contrast-upsell-modal.svg
Comment (dougfabris) [inline apps/meteor/public/images/high-contrast-upsell-modal.svg:1]: isn't lightest to use `.png ` or `.jpeg`?

**Review by dougfabris — CHANGES_REQUESTED** [review]

File: apps/meteor/client/views/account/themes/ThemePage.tsx (hunk 1 above)
Comment (juliajforesti) [reply apps/meteor/client/views/account/themes/ThemePage.tsx:18]: yep, will do that

File: apps/meteor/public/images/high-contrast-upsell-modal.svg
Comment (juliajforesti) [reply apps/meteor/public/images/high-contrast-upsell-modal.svg:1]: hum right, i'll change that

**Review by ggazzo — APPROVED** [review]

**Pull request merged**